package.json

```diff
@@ -24,6 +24,7 @@
     "@mui/x-data-grid": "^5.10.0",
     "@mui/x-data-grid-pro": "^5.10.0",
     "clsx": "^1.1.1",
+    "docusaurus-gtm-plugin": "^0.0.2",
```

Comment:
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag is this different? Seems like the official one?

Reply:
It seems to me like there's a difference between "google site tag" and a "google tag manager". They each take different IDs assigned by Google. We already have gtag implemented and it didn't seem to do the job for @cmo-pomerium, maybe he can weigh in further.